src/css/style.css

```diff
@@ -0,0 +1,190 @@
+body {
+  display: flex;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+
+.project-create-btn {
+  background: #008cba;
+  color: #fff;
+  font-size: 1rem;
+  border-radius: 1.6rem;
+  border: none;
+  padding: 0.6rem;
+}
+
+.create-btn {
+  padding-right: 1rem;
+}
+
+.fa-plus { margin-right: 1rem; }
+
+main {
+  background-image: url('../images/bg.jpg');
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 90%;
+  height: 100vh;
+}
+
+.new-project-input {
+  border-radius: 6px;
+  height: 2rem;
+  background: #efefef;
+  width: 100%;
+  position: relative;
+  top: 2rem;
+  border: 1px solid#efefef;
+  outline: none;
+}
+
+article {
+  position: relative;
+  top: 8rem;
+  left: 8rem;
+  width: 30%;
+  height: 8rem;
+  background: white;
+  border-radius: 6px;
+  padding: 8rem;
+  display: none;
+}
+
+.new-project-btns button {
+  width: 49%;
+  height: 50%;
+  position: relative;
+  top: 4rem;
+  border: none;
+  background: white;
+  border-right: 2px solid #efefef;
+  text-transform: uppercase;
+  box-shadow: 1px 1px gray;
+  outline: none;
+}
+
+.new-project-btns button:hover {
+  background: #efefef;
+}
+
+.task.completed {
+  text-decoration: line-through;
+  color: gray;
+}
+
+.task.completed span {
+  text-decoration: line-through;
+  color: gray;
+}
+
+aside {
+  padding: 4rem;
+  background: white;
+  width: 100%;
+  height: 100vh;
+  position: relative;
+  top: 4rem;
+  margin: 1rem;
+}
+
+.form-container {
+  padding: 1rem;
+}
+
+form input {
+  border: 1px solid  #efefef;
+  background: rgb(240, 240, 240);
+  height: 2.2rem;
+  outline: none;
+}
+
+#datetime {
+  width: 9.6rem;
+}
+
+/* input[type='date']::before {
+  content: attr(placeholder);
+} */
+
```

Comment:
- [ ] Please remove commented lines of code.

Reply:
-I have made the delete task to work properly
-I have made the priority to work both high and low by choosing the radio button